### PR TITLE
2024 logbooks

### PR DESCRIPTION
**Pull request by JoschD**

### CODE REVIEW

File: docs/resources/logbook/2024_sps.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Timeline of SPS optics measurements in 2023
```
Comment (fsoubelet) [inline docs/resources/logbook/2024_sps.md:1]: Should be 2024

File: docs/resources/logbook/2024_leir.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+# Timeline of LEIR optics measurements in 2023
```
Comment (fsoubelet) [inline docs/resources/logbook/2024_leir.md:2]: Should be 2024

File: scripts/shift_calculations.py [hunk 3]
```diff
@@ -350,10 +350,22 @@ def plot_results(parts, title: str = "", output_path: Union[str, Path] = None):
...
-    shift_m = manual_shifts("/mnt/volume/jdilly/projects/pylhc.github.io/docs/resources/logbook/2023_lhc.md")
-    plot_results(shift_m, title="OMC Shifts LHC 2023")
+    shift_m = manual_shifts(logbook_dir / "2023_lhc.md")
+    plot_results(shift_m, title="OMC Shifts LHC 2023", output_path="lhc_2023_shifts.pdf")
+    
+    shift_m = manual_shifts(logbook_dir / "2023_ps.md")
+    plot_results(shift_m, title="OMC Shifts PS 2023", output_path="ps_2023_shifts.pdf")
+    
+    shift_m = manual_shifts(logbook_dir / "2023_psb.md")
+    plot_results(shift_m, title="OMC Shifts PSBooster 2023", output_path="psb_2023_shifts.pdf")
```
Comment (fsoubelet) [inline scripts/shift_calculations.py:369]: Do we want to keep the 2023 year hardcoded here? If not we could also add the 2024 off days to the `CERN_HOLIDAYS` dict (line 96 onwards).

Comment (JoschD) [reply scripts/shift_calculations.py:369]: for now is fine I think. We can change whenever

**Review by fsoubelet — APPROVED** [review]

**Pull request merged**